src/display/annotation_layer.js

```diff
@@ -76,6 +76,15 @@ AnnotationElementFactory.prototype =
         switch (fieldType) {
           case 'Tx':
             return new TextWidgetAnnotationElement(parameters);
+          case 'Btn':
+            if (!parameters.data.pushbutton) {
```

Comment:
Is this warning really necessary though, since it's going to be printed in the console regardless if forms are enabled or not?
Given that forms support is still some ways from being implemented, is it necessary to bother users about this?